rules_building_block/execution_aws_lambda_function_updated.toml

```diff
@@ -0,0 +1,71 @@
+[metadata]
+creation_date = "2024/04/20"
+integration = ["aws"]
+maturity = "production"
+min_stack_comments = "AWS integration breaking changes, bumping version to ^2.0.0"
+min_stack_version = "8.9.0"
+updated_date = "2024/04/20"
+bypass_bbr_timing = true
+
+[rule]
+author = ["Elastic"]
+building_block_type = "default"
+description = """
+Identifies when an AWS Lambda function is created or updated. AWS Lambda lets you run code without provisioning or
+managing servers. Adversaries can create or update Lambda functions to execute malicious code, exfiltrate data, or
+escalate privileges. This is a [building block
+rule](https://www.elastic.co/guide/en/security/current/building-block-rule.html) that does not generate alerts, but
```

Comment:
Technically BBRs do generate alerts right?

Reply:
They generate events in the signal index if I recall.